whakarere.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
```

Comment:
flathub is not meant to host the upstream software which this looks to be.

Reply:
i have removed the files from the commit